packages/block-serialization-default-parser/test/index.js

```diff
@@ -17,7 +17,7 @@ import {
  */
 import { parse } from '../src';
 
-describe( 'block-serialization-default-parser-js', jsTester( parse ) ); // eslint-disable-line jest/valid-describe
+describe( 'block-serialization-default-parser-js', jsTester( parse ) ); // eslint-disable-line jest/valid-describe-callback
```

Comment:
The rule name changed: https://github.com/jest-community/eslint-plugin-jest/releases/tag/v25.0.0